scripts/build_env.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+function buildEnv { 
+    defaultSecretKey="$(hexdump -vn16 -e'4/4 "%08X" 1 "\n"' /dev/urandom | base64)"
```

Comment:
테스트해보니 줄바꿈 문자도 들어가는 것 같네요...
```suggestion
    defaultSecretKey="$(hexdump -vn16 -e'4/4 "%08X"' /dev/urandom | base64)"
```